README.md

```diff
@@ -9,84 +9,147 @@ A [Test Kitchen][kitchenci] Driver for Amazon EC2.
 This driver uses the [aws sdk gem][aws_sdk_gem] to provision and destroy EC2
 instances. Use Amazon's cloud for your infrastructure testing!
 
+## Initial Setup
+
+To get started, you need to install the software and set up your credentials and SSH key. Some of these steps you have probably already done, but we include them here for completeness.
+
+1. Install the latest ChefDK and put it in your path.
```

Comment:
The TK maintainers are trying to ensure the project is agnostic to the use case. So, we should probably change this line to add "or ensure the test-kitchen gem is installed on your workstation"?